angular/js/controllers/submitController.js

```diff
@@ -45,6 +45,7 @@ angular.module('nuswhispersApp.controllers')
     $scope.submitConfession = function () {
         $scope.form.submitButtonDisabled = true;
         $scope.confessionData.categories = $scope.form.selectedCategoryIDs;
+        $scope.confessionData.token = localStorageService.get('confessionToken');
```

Comment:
Not sure if we should try to obfuscate the name of this string to make it less obvious that we're tracking people lol.